schemas/snuba-eap-mutations.v1.schema.json

```diff
@@ -0,0 +1,55 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$ref": "#/definitions/Main",
+  "definitions": {
+    "Main": {
+      "type": "object",
+      "title": "eap_mutation",
+      "properties": {
+        "filter": {
+          "type": "object",
+          "additionalProperties": false,
+          "properties": {
+            "organization_id": {
+              "type": "integer"
+            },
+            "_sort_timestamp": {
+              "type": "integer"
+            },
+            "trace_id": {
+              "$ref": "#/definitions/UUID",
+              "description": "The trace ID is a unique identifier for a trace. It is a 16 byte hexadecimal string."
+            },
+            "span_id": {
+              "type": "integer"
```

Comment:
@untitaker Had some questions here -- I was comparing against the spans topic schema, and I think `span_id` is a String? I noticed a couple of other examples like this too

Reply:
you are right. please change this in the schema. it should not be an int at all, that's a mistake in the code